packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -1327,48 +1327,40 @@ void main() {
         );
     }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
 
-    testWidgets('buildSimpleRefreshIndicator dark mode', (WidgetTester tester) async {
```

Comment:
Should this still be tested in dark mode one way or another?

Reply:
Excellent question. We don't do anything specific depending on what mode we're in... not sure if the standard practice is to always do a light and dark mode test, or only if the widget actually does something different. Do you know if there's any specific guidance one way or another?

Reply:
No, it's not necessary if the change doesn't have any special dark mode behavior. But why was this test replaced?  Could you keep the original test and add your new one, or does the original test conflict with the new behavior?

Reply:
Ah I see your response to @Piinks above asking about the same thing.  This is fine as-is since the dark mode specific behavior was removed. 👍